### PR TITLE
Cleanup theming

### PR DESCRIPTION
**Pull request by Renzo-Olivares**
This change cleans up the light and dark theme properties in `app.dart`. Removes any dependency on `navigationRailTheme` since there is no longer a navigation rail in this project.

### CODE REVIEW

File: lib/bottom_drawer.dart [hunk 1]
```diff
@@ -49,8 +49,7 @@ class BottomDrawer extends StatelessWidget {
               child: Text(
                 'FOLDERS',
                 style: theme.textTheme.caption.copyWith(
-                  color:
-                      theme.navigationRailTheme.unselectedLabelTextStyle.color,
+                  color: ReplyColors.white50.withOpacity(0.64),
```
Comment (rami-a) [inline lib/bottom_drawer.dart:52]: Could this come from the color scheme?

File: lib/home.dart [hunk 2]
```diff
@@ -642,15 +642,14 @@ class _BottomDrawerDestinations extends StatelessWidget {
                 ),
                 color: destination.index == selectedIndex
                     ? theme.colorScheme.secondary
-                    : theme.navigationRailTheme.unselectedLabelTextStyle.color,
+                    : ReplyColors.white50.withOpacity(0.64),
```
Comment (rami-a) [inline lib/home.dart:645]: Could this (and below) come from the color scheme?

**Review by rami-a — APPROVED** [review]
LGTM

File: lib/home.dart (hunk 2 above)
Comment (Renzo-Olivares) [reply lib/home.dart:645]: replied in other comment.

**Review by clocksmith — APPROVED** [review]

**Pull request merged**